src/backend/src/controllers/design-review.controllers.ts

```diff
@@ -1 +1,17 @@
-export default class DesignReviewController {}
+import { User } from '@prisma/client';
+import { Request, Response, NextFunction } from 'express';
+import { getCurrentUser } from '../utils/auth.utils';
+import DesignReviewService from '../services/design-review.services';
+
+export default class DesignReviewController {
+  static async deleteDesignReview(req: Request, res: Response, next: NextFunction) {
+    try {
+      const drId: string = req.params.designReviewId;
+      const user: User = await getCurrentUser(res);
+      await DesignReviewService.deleteDesignReview(user, drId);
```

Comment:
you should return the deleted design review from the service method then return it in the response